tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist=py27,flake8,py3
+
+[testenv]
+deps =
+    -r{toxinidir}/requirements.lint.txt
+    nose
+    coverage
+commands = nosetests -v {posargs} tests.py
+
+
+[testenv:flake8]
+commands = flake8 src/
```

Comment:
I got an error in *flake8* running `tox`. I can run it fine manually `flake8 src/`. Are you able to run it using *tox*?

Reply:
I ran ` tox -eflake8`

Or you can run `tox` and run all the tests.

Reply:
When I run `tox -eflake8` I get:

```
GLOB sdist-make: /home/kashif/Code/nose-socket-whitelist/setup.py
flake8 create: /home/kashif/Code/nose-socket-whitelist/.tox/flake8
flake8 installdeps: -r/home/kashif/Code/nose-socket-whitelist/requirements.lint.txt, nose, coverage
flake8 inst: /home/kashif/Code/nose-socket-whitelist/.tox/dist/nose-socket-whitelist-1.0.0.zip
flake8 installed: coverage==4.4.2,flake8==1.4,future==0.16.0,nose==1.3.7,nose-socket-whitelist==1.0.0
flake8 runtests: PYTHONHASHSEED='98698230'
flake8 runtests: commands[0] | flake8 src/
Traceback (most recent call last):
  File "/home/kashif/Code/nose-socket-whitelist/.tox/flake8/bin/flake8", line 5, in <module>
    main()
  File "/home/kashif/Code/nose-socket-whitelist/.tox/flake8/lib/python3.6/site-packages/flake8/run.py", line 48, in main
    options, args = pep8.process_options()
  File "/home/kashif/Code/nose-socket-whitelist/.tox/flake8/lib/python3.6/site-packages/flake8/pep8.py", line 1672, in process_options
    options.physical_checks = find_checks('physical_line')
  File "/home/kashif/Code/nose-socket-whitelist/.tox/flake8/lib/python3.6/site-packages/flake8/pep8.py", line 1106, in find_checks
    for name, function in globals().items():
RuntimeError: dictionary changed size during iteration
ERROR: InvocationError: '/home/kashif/Code/nose-socket-whitelist/.tox/flake8/bin/flake8 src/'
```

Any idea why that is?

Reply:
Looks like it's because the version of flake8 specified in `requirements.lint.txt` is really old. We should update that to `flake8==3.5.0`. You can either add it in here or I'll add it after you merge.